.travis.yml

```diff
@@ -1,6 +1,7 @@
 branches:
   only:
   - master
+  - /^it\/.*$/
```

Comment:
Up to now only the master branch and pull request branches with base branch "master" was taken into account. But now we have also a branch pattern for triggering integration tests. Hence we have to listen also for changes in that pattern. --> is the pattern ìt/'`fine?